include/odp/api/spec/traffic_mngr.h

```diff
@@ -794,20 +794,26 @@ typedef enum {
  */
 typedef struct {
 	/** The committed information rate for this shaper profile.  The units
-	 * for this integer are always in bits per second. */
+	 * for this integer is in bits per second when packet_mode is
+	 * not TRUE while packets per second when packet mode is TRUE.
+	 */
 	uint64_t commit_bps;
 
 	/** The peak information rate for this shaper profile.  The units for
-	 * this integer are always in bits per second. */
+	 * this integer is in bits per second when packet_mode is
+	 * not TRUE while in packets per second when packet mode is TRUE.
+	 */
 	uint64_t peak_bps;
```

Comment:
Here we end up with *_bps parameters that are actually packets per second, not bits per second, if packet_mode is true, which is pretty horrible. At the very least the _bps postfix should be removed from these parameter names, but I'm not sure if we want to do that kind of incompatible change. @psavol should comment.

Reply:
Ack, missed that. I can make it _rate and create a union of older deprecated name and new name if it is fine.